Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -333,24 +337,6 @@ def auto_detect_indicator_type(indicator_value):
     return None
 
 
-# ===== Fix fetching credentials from vault instances =====
```

Comment:
@liorblob is it save to remove this ?

Reply:
I don't think so... not sure it was handled on server side

Reply:
@liorblob From what @yaakovi and I saw this was up to 4.5. Isn't that the case?

Reply:
what do you mean by "up to 4.5"?

Reply:
This code was added in CommonServerPython4.1 as can be [seen here](https://github.com/demisto/content/pull/3361/files).

Reply:
That file doesn't exist anymore... so if you remove that it will not work for versions under 4.5

Reply:
@Itay4 are we safe on that front because of the feature branch?

Reply:
@orlichter1 yes, 4.5 and below will still get this code via the feature branch.
@liorblob as long as this is fixed for version 5.0.0 on the server side - we can remove it from here

Reply:
should be ok then